src/components/AddressInputPanel/index.tsx

```diff
@@ -190,16 +190,15 @@ export default function AddressInputPanel({
               <TYPE.black color={theme.text2} fontWeight={500} fontSize={14}>
                 Recipient
               </TYPE.black>
-
-              {data.name ? (
-                <Link href={getEtherscanLink(chainId, data.name, 'address')} style={{ fontSize: '14px' }}>
-                  (View on Etherscan)
-                </Link>
-              ) : (
-                <Link href={getEtherscanLink(chainId, data.address, 'address')} style={{ fontSize: '14px' }}>
-                  (View on Etherscan)
-                </Link>
-              )}
+              {data.name ||
+                (data.address && (
```

Comment:
the parens are wrong here btw, let's try to be more careful. i'll fix in in my PR, so no need to do anything.